sql_instruction.txt

```diff
@@ -0,0 +1,5 @@
+mysql -u colte_db -p
```

Comment:
What are these instructions for? If they're intended for the public repo they may need some more context.